vpdq/cpp/hashing/filehasher.cpp

```diff
@@ -96,10 +97,22 @@ bool hashVideoFile(
             "%s: failed to hash frame buffer. Frame width or height smaller than minimum hashable dimension. %d.\n",
             argv0,
             fno);
+        pclose(inputFp);
         return false;
       }
+
+      // Avoid divide by zero when calculating timestamp
+      // The timestamp will equal the frame number of the hash
+      // if framesPerSec <= 0. This is common on very short videos.
+      double timestamp = 0;
+      if (framesPerSec > 0) {
+        timestamp = (double)fno / framesPerSec;
+      } else {
+        timestamp = fno;
```

Comment:
Can you tell me a little bit more about the types of files that generate this? 

By any chance are they all single frame gifs or something like that?

I'm worried that it will distort the meaning of timestamp if multi-frame 0 fps items exist, but I don't know what a multi-frame 0 fps video might be other than invalid :P 

If they are all single frame only, we can make it more clear what edge case we are catching and crash / except for multi-frame 0fps since it's not going to produce a valid vPDQ hash as we've defined it.